src/constants.ts

```diff
@@ -48,6 +48,11 @@ export class Icons {
     }
 }
 
+
+export enum GlyphChars {
+    Check = '\u2713'
```

Comment:
Just for information, the idea of GlyphChars comes from the vscode-gitlens https://github.com/eamodio/vscode-gitlens/blob/7f549c66e7ade7c569f5acd906d170646d7ab5d9/src/views/nodes/branchNode.ts#L392